evaluations/README.md

```diff
@@ -2,40 +2,44 @@
 
 This folder aims at collecting evaluations of forecasting models. The goal is to make reproducibility and comparison easier by versioning the code producing dataset as well as the model and evaluation code.
 
-
 ## mean_wQuantileLoss
 
 estimator | electricity | exchange_rate | m4_daily | m4_hourly | m4_monthly | m4_quarterly | m4_weekly | m4_yearly | solar-energy | traffic
 ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ----
-DeepAREstimator | 0.085 | 0.009 | 0.025 | 0.033 | 0.115 | 0.087 | 0.048 | 0.128 | 0.398 | 0.126
-MQCNNEstimator | 0.102 | 0.016 | 0.028 | 0.065 | 0.123 | 0.089 | 0.060 | 0.124 | 0.627 | 0.274
-NPTSPredictor | 0.144 | 0.021 | 0.145 | 0.048 | 0.233 | 0.255 | 0.297 | 0.355 | 0.825 | 0.179
-RForecastPredictor |  | 0.008 |  |  |  |  |  |  |  | 
-SeasonalNaivePredictor | 0.139 | 0.011 | 0.028 | 0.048 | 0.146 | 0.119 | 0.063 | 0.161 | 1.000 | 0.251
-SimpleFeedForwardEstimator | 0.095 | 0.009 | 0.023 | 0.044 | 0.119 | 0.085 | 0.053 | 0.127 | 0.436 | 0.213
+DeepAREstimator | 0.050 | 0.023 | 0.025 | 0.033 | 0.115 | 0.087 | 0.048 | 0.128 | 0.398 | 0.126
```

Comment:
can we wait that the results for methods are complete before merging? (otherwise I dont have comments beside that this is great to have those!)

Reply:
Sure, I’ll update the PR once all R jobs are done

Reply:
@geoalgo see commit 4f0c251, that's as much as we can get, all other jobs did not finish before 24 hours and were stopped, so I would just skip them

Reply:
Yes, it makes sense, one thing is that we should them with the parallelizer Valentin wrote.